querybook/webapp/components/DataDocViewersList/DataDocViewersList.scss

```diff
@@ -1,13 +1,9 @@
 .DataDocViewersList {
     width: 400px;
-    padding: 0.5rem;
 
-    > div {
-        padding: 0.5rem;
-    }
-
-    .public-row-switch {
-        padding-bottom: 1rem;
+    > div:not(:last-child) {
```

Comment:
hmm I don't think its a good idea to apply a per div style here since the .DataDocViewersList's children isn't really a list

Reply:
it's not a list but it's not going to change. i want to keep this